Gemfile.lock

```diff
@@ -76,7 +76,7 @@ GEM
       activerecord (>= 3.2, < 6.0)
       rake (~> 10.4)
     arel (3.0.3)
-    bootstrap-sass (2.3.1.2)
+    bootstrap-sass (2.3.1.3)
```

Comment:
Can't we move to [2.3.2.2](https://rubygems.org/gems/bootstrap-sass/versions/2.3.2.2)? Commit message comments as to why not would be worth doing.